### PR TITLE
Fix benchmarking with sbt 1.0, adapt InfluxDB

### PR DESCRIPTION
**Pull request by Duhemm**
We'll still have to figure out authentication with the InfluxDB server, but I guess we'll deal with that once we get a real instance.

### CODE REVIEW

File: infrastructure/src/main/java/scala/bench/GitFactory.java [hunk 1]
```diff
@@ -11,7 +11,12 @@
 public class GitFactory {
     public static Repository openGit() {
         Config conf = ConfigFactory.load();
-        String gitLocalDir = conf.getString("git.localdir");
+        String gitLocalDir = System.getProperty("git.localdir");
+
+        if (gitLocalDir == null) {
```
Comment (jvican) [inline infrastructure/src/main/java/scala/bench/GitFactory.java:16]: 👍

File: infrastructure/src/main/resources/reference.conf [hunk 2]
```diff
@@ -1,11 +1,9 @@
 influx {
-  url = "https://scala-ci.typesafe.com/influx/"
-  user = "scala"
+  url = "http://localhost:8086"
```
Comment (jvican) [inline infrastructure/src/main/resources/reference.conf:2]: Will this be available from the outside? Shouldn't we use here a URL that Fabien gives us?

**Review by jvican — APPROVED** [review]
Looks awesome, just one request.

File: infrastructure/src/main/resources/reference.conf (hunk 2 above)
Comment (Duhemm) [reply infrastructure/src/main/resources/reference.conf:2]: Not yet. There are InfluxDB and Grafana instances on our server, but it is not open from outside EPFL. Let's see what we'll do...

**Pull request merged**